manifests/init.pp

```diff
@@ -384,11 +384,13 @@
       encoding            => 'UTF-8',
       locale              => 'en_US.UTF-8',
       manage_package_repo => true,
+      version             => $postgresql_version,
     }
     include postgresql::params
     $default_data_dir = $postgresql::params::datadir
     $default_bin_dir = $postgresql::params::bindir
-    package { $postgresql::params::server_package_name:
+    $postgresql_package_name = $postgresql::params::server_package_name
+    package { $postgresql_package_name:
```

Comment:
Could we instead have a static resource name and use namevar to set the actual name? This makes tracing the resource through the code and tests much easier since it is no longer variable.

Reply:
Updated